addon/testing.js

```diff
@@ -0,0 +1,73 @@
+import {get} from '@ember/object';
+import {yieldableSymbol, YIELDABLE_CONTINUE} from './yieldables';
+import {TimeoutPromise} from './-timeout';
+
+let TEST_TIMERS = {};
```

Comment:
FYI, when using the new test API, you can `import { getContext } from '@ember/test-helpers'` to get the current test context object (if a test is running), i.e. `this` in functions passed to `test()` and `beforeEach()` and whatnot. If you prefer to scope any of this state to "the currently running test" you could store it on that object instead of module-scoping it. I guess you'd want fallbacks in case the user is running against the test API, and I'm not sure what you want to do if a test isn't currently running...anyway, not certain this is a direction you want to go, but figured I'd mention it in case.

Reply:
Oh, I guess this code only runs when `Ember.testing` is true, so you at least don't have to worry about the using-new-api-but-test-not-running case.

Reply:
I def didn't know about getContext(), that's a much better place for it :)